lib/vrl/stdlib/src/parse_json.rs

```diff
@@ -115,29 +127,13 @@ mod tests {
         parses {
             args: func_args![ value: r#"{"field": "value"}"# ],
             want: Ok(value!({ field: "value" })),
-            tdef: TypeDef::new()
-                .fallible()
-                .bytes()
-                .add_boolean()
-                .add_integer()
-                .add_float()
-                .add_null()
-                .add_array_mapped::<(), Kind>(map! { (): Kind::all() })
-                .add_object::<(), Kind>(map! { (): Kind::all() }),
+            tdef: type_def(),
```

Comment:
I prefer to have at least one of these spell out the exact type expected.

The reason for this is, if `type_def()` is badly implemented, we would never see it fail. At least with one test defining the expectation manually, you see the test fail when you update the function, and then consciously update the test to match your expectations (ideally, taking a more TDD-like approach, you'd update the test first, to match your expectations, and then update `type_def()` to match those expectations and get the test case green).